contactsummary/scss/contactsummary.scss

```diff
@@ -1,6 +1,10 @@
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables";
 @import "SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/variables";
 
+#bootstrap-theme {
```

Comment:
I think that should go to Shoreditch, no?

Reply:
No, these elements and components are only for the contact summary. I'm adding `#bootstrap-theme` here to avoid using `!important` inside of the partials. Since some rules come from Shoreditch such as this:

```sass
#bootstrap-theme ul {
  padding-left: 30px;
}
```

To overwrite that you'll either need another ID or use !important. I Opted to use the #bootstrap-theme wrapper for these cases where you want to overwrite shoreditch.